libraries/libformula-ui/src/test/java/org/pentaho/openformula/ui/FormulaDocumentTest.java

```diff
@@ -12,14 +12,21 @@
 * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 * See the GNU Lesser General Public License for more details.
 *
-* Copyright (c) 2002-2013 Pentaho Corporation..  All rights reserved.
+* Copyright (c) 2002-2018 Hitachi Vantara..  All rights reserved.
 */
 
 package org.pentaho.openformula.ui;
 
 import junit.framework.TestCase;
+import org.pentaho.openformula.ui.model2.FormulaClosingParenthesisElement;
 import org.pentaho.openformula.ui.model2.FormulaDocument;
+import org.pentaho.openformula.ui.model2.FormulaElement;
+import org.pentaho.openformula.ui.model2.FormulaOpenParenthesisElement;
+import org.pentaho.openformula.ui.model2.FormulaOperatorElement;
+import org.pentaho.openformula.ui.model2.FormulaParser;
 import org.pentaho.openformula.ui.model2.FormulaRootElement;
+import org.pentaho.openformula.ui.model2.FormulaSemicolonElement;
+import org.pentaho.openformula.ui.model2.FormulaTextElement;
 
 import javax.swing.text.BadLocationException;
 
```

Comment:
In the original PR there's a static import for "org.pentaho.openformula.ui.model2.FormulaParser.parseText".
Is it missing or simply unnecessary (wingman runned fine without it)?